src/pages/Study.tsx

```
@@ -0,0 +1,125 @@
+    <div css={Container}>
+      <div css={Header}>
+        <img css={BackButton} src={backIcon} onClick={() => navigate(-1)} />
+        <p css={DayText}>{day}일차</p>
+      </div>
+      <div css={Content}>
+        <p css={Title}>학습하기</p>
+        <Speaker text={selectedTopic} />
+        <div css={ButtonWrapper}>
+          {Topic.map((index) => (
+            <div css={TopicButton} onClick={() => setSelectedTopic(index.name)}>
+              <img css={Icon} src={index.icon} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)